rasterio/_io.pyx

```diff
@@ -4,29 +4,27 @@
 from __future__ import absolute_import
 
 import logging
-import math
 import os
 import os.path
-import sys
 import warnings
 
-from libc.stdlib cimport malloc, free
 import numpy as np
 cimport numpy as np
 
-from rasterio cimport _base, _gdal, _ogr, _io
-from rasterio._base import (
-    crop_window, eval_window, window_shape, window_index, tastes_like_gdal)
-from rasterio._drivers import driver_count, GDALEnv
-from rasterio._err import CPLErrors, GDALError, CPLE_OpenFailed
+from rasterio cimport _base
```

Comment:
@geowurster is there a reason for splitting these out onto individual lines?  PEP8 seems to allow this pattern.